### PR TITLE
feat: add boost assert

### PR DESCRIPTION
**Pull request by wep21**
- add [boost.assert](http://boost.org/libs/assert)
- need skip url sanity check

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (boost.assert) have been updated in this PR. Please review the changes.

File: modules/boost.assert/1.83.0/presubmit.yml
Comment (wep21) [reply modules/boost.assert/1.83.0/presubmit.yml:16]: boost.assert tests depend on boost.core, but boost.core depends on boost.assert. Is it okay to add tests after merging this PR once?

Comment (meteorcloudy) [reply modules/boost.assert/1.83.0/presubmit.yml:16]: Sure!

**Review by meteorcloudy — APPROVED** [review]

**Pull request merged**
